src/Apps/__stories__/OrderApp.story.tsx

```diff
@@ -28,8 +46,24 @@ const Router = props => (
   />
 )
 
+const RouterWithPickupOrder = props => (
+  <StorybooksRouter
```

Comment:
Can we adjust this component to accept a `mock` option so we don't have to duplicate here?